sector-builder/src/scheduler.rs

```diff
@@ -66,7 +66,25 @@ impl Scheduler {
         scheduler_rx: mpsc::Receiver<SchedulerTask<U>>,
         worker_tx: mpsc::Sender<WorkerTask<U>>,
         mut m: SectorMetadataManager<T, S>,
-    ) -> Scheduler {
+    ) -> Result<Scheduler> {
+        // If a previous instance of the SectorBuilder was shut down mid-seal,
+        // its metadata store will contain staged sectors who are still
+        // "Sealing." If we do have any of those when we start the Scheduler,
+        // we should immediately restart sealing.
+        //
+        // For more information, see rust-fil-sector-builder/17.
+        let protos: Result<Vec<SealTaskPrototype>> = m
+            .get_staged_sectors(Some(SealStatus::Sealing))
```

Comment:
maybe rename the method to sth like `get_staged_sector_filtered`, right now it's a bit surprising this takes an argument.

Reply:
sounds good